docs/usage.rst

```diff
@@ -6,86 +6,82 @@ To use TIA Toolbox in a project::
 
     import tiatoolbox
 
-
-----------
-Dataloader
-----------
-.. automodule:: tiatoolbox.dataloader
-
-^^^^^^^^^^^^^^^^^^^^
-dataloader.wsireader
-^^^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^^^^^^^
+Reading WSI Image Data
+^^^^^^^^^^^^^^^^^^^^^^
 
 .. currentmodule:: tiatoolbox.dataloader.wsireader
 .. autoclass:: WSIReader
-    :members: read_region, slide_thumbnail, save_tiles
+    :members: read_rect, read_bounds, read_region, slide_thumbnail, save_tiles
```

Comment:
are we changing `slidethumbnail` to `get_thumbnail`?

Reply:
Can do in this PR or separate. I'm happy either way.

Reply:
Done in commit https://github.com/TIA-Lab/tiatoolbox/pull/40/commits/674d0851d6cf6b9c8e3e431ed5aa269d360c199e